packages/backend/src/peripherals/database/types.ts

```diff
@@ -106,6 +107,17 @@ declare module 'knex/types/tables' {
     eth_address: string
   }
 
+  type ForcedTransactionEventRow = {
```

Comment:
As a general note in this PR. The rest of the codebase follows the style of declaring shapes as interfaces, but here and elsewhere in this PR type aliases are used. It would be cool to keep a consistent style

Reply:
Good point - changed.